boot-core/src/daemon/state.rs

```diff
@@ -147,29 +175,61 @@ impl DaemonState {
             lcd_url: None,
             fcd_url: None,
         };
+
+        // check validity
         state.check_file_validity();
+
+        // finish
         Ok(state)
     }
 
+    // maybe we shold rename this?
     pub fn check_file_validity(&self) {
-        let file = File::open(&self.json_file_path).unwrap_or_else(|_| {
-            let file = File::create(&self.json_file_path).unwrap();
-            serde_json::to_writer_pretty(&file, &json!({})).unwrap();
-            File::open(&self.json_file_path).unwrap()
-        });
-        log::info!("Opening daemon state at {}", self.json_file_path);
```

Comment:
Would be nice to still have this logged!

Reply:
uops, adding that back

Reply:
I don't see it in here yet?